tests/components/reolink/test_init.py

```diff
@@ -39,6 +40,11 @@
 pytestmark = pytest.mark.usefixtures("reolink_connect", "reolink_platforms")
 
 
+async def test_wait(*args, **key_args):
+    """Ensure a mocked function takes a bit of time to be able to timeout in test."""
+    await asyncio.sleep(0.01)
```

Comment:
Ideally this is sleep 0 to only yield

Reply:
I just tested with sleep(0) and that also works, so changed it to sleep(0)